### ISSUE TITLE
Fix scaladoc warnings

### ISSUE THREAD

**Issue by vivienlacourba** [w3c/validator-suite#324]
Please fix the warnings below which are reported during the scaladoc generation:
Source: https://atkins.w3.org/jenkins/job/validator-suite/935/consoleFull

```
[info] Generating Scala API documentation for main sources to /u/jenkins/workspace/validator-suite/target/scala-2.10/api...
model contains 412 documentable templates
[warn] /u/jenkins/workspace/validator-suite/app/views/newOneTimeJob.scala.html:23: Tag '@if(messages.size' is not recognised
[warn] /u/jenkins/workspace/validator-suite/app/views/newOneTimeJob.scala.html:46: Tag '@if(messages.size' is not recognised
[warn] /u/jenkins/workspace/validator-suite/app/views/forms/oneTimeJobForm.scala.html:184: Variable 50 undefined in comment for object oneTimeJobForm in object oneTimeJobForm
[warn] /u/jenkins/workspace/validator-suite/app/views/forms/oneTimeJobForm.scala.html:23: unbalanced or unclosed heading
[warn]                     Purchase a new W3C Validator Suite<span class="tm">™</span> <strong>One Time</strong> service job.
[warn] ^
[warn] /u/jenkins/workspace/validator-suite/app/views/forms/oneTimeJobForm.scala.html:347: Variable 50 undefined in comment for method apply in object oneTimeJobForm
[warn] /u/jenkins/workspace/validator-suite/app/views/forms/oneTimeJobForm.scala.html:186: unbalanced or unclosed heading
[warn] /u/jenkins/workspace/validator-suite/app/assertor/UnicornFormatAssertor.scala:11: Could not find any member to link for "scala.io.Source".
[warn] /** An Assertor that reads [[http://code.w3.org/unicorn/wiki/Documentation/Run/Response ObservationResponse]]s from [[scala.io.Source]]s
[warn] ^
[warn] /u/jenkins/workspace/validator-suite/app/actor/RunAction.scala:19: Tag '@token' is not recognised
[warn] /**
[warn] ^
[warn] /u/jenkins/workspace/validator-suite/app/model/User.scala:159: Variable 2a undefined in comment for method rootPassword in object User
[warn]   def rootPassword: String = {
[warn]       ^
[warn] /u/jenkins/workspace/validator-suite/app/model/Run.scala:438: Tag '@returns' is not recognised
[warn]   /** apply one event to the current Run. This targets both the
[warn]   ^
[warn] 10 warnings found
[info] Scala API documentation generation successful.
[success] Total time: 39 s, completed Oct 21, 2013 4:04:02 PM
```
